src/state.rs

```diff
@@ -65,4 +65,8 @@ impl Parser {
         parser.nextToken();
         parser.parseExpression()
     }
+
+    pub fn tokenizer(input: String, options: Option<option::Options>) -> node::Node {
+        Parser::new(options.unwrap_or_default(), input)
+    }
```

Comment:
That's... it?

Reply:
https://github.com/acornjs/acorn/blob/master/acorn/src/state.js#L121

Reply:
wait, how does `Parser::new` return `node::Node`?

Reply:
How is your `parse` and `parseExpressionAt` functions returning `node::Node` ?